tests/test_run.py

```diff
@@ -100,17 +102,29 @@ def test_run_script_full(provider):
                 "--create",
                 "--api-key",
                 api_key,
+                "-o",
+                str(output_file),
+                "-e",
+                str(error_file),
+                "--dry-run",
```

Comment:
No dry run. This is intended to actually hit the providers as a form of integration test.